c10/core/WrapDimMinimal.h

```diff
@@ -28,7 +28,7 @@ T _maybe_wrap_dim(T dim, T dim_post_expr, bool wrap_scalar = true) {
   }
   // Check edge-cases out-of-line (wrapping scalars and out-of-bounds errors)
   return c10::detail::maybe_wrap_dim_slow<T>(
-      std::move(dim), std::move(dim_post_expr), wrap_scalar);
```

Comment:
Is the really issue here the some of the std::move should be std::forward and that's what the static analyzer in complaining about?